src/main/java/com/atlassian/jira/cloud/jenkins/ping/JenkinsAppPingRequest.java

```diff
@@ -0,0 +1,11 @@
+package com.atlassian.jira.cloud.jenkins.ping;
+
+import com.atlassian.jira.cloud.jenkins.common.client.JenkinsAppRequest;
+
+/** A request that we can send to the Jenkins app to check the connection. */
+public class JenkinsAppPingRequest extends JenkinsAppRequest {
+
+    public JenkinsAppPingRequest() {
+        super(RequestType.PING);
+    }
+}
```

Comment:
New request type for the "ping" event that's basically just a "hello world" handshake.